cmd/regctl/image.go

```diff
@@ -1383,51 +1386,78 @@ func (imageOpts *imageCmd) runImageCreate(cmd *cobra.Command, args []string) err
 }
 
 func (imageOpts *imageCmd) runImageExport(cmd *cobra.Command, args []string) error {
+	var output string
+	// Legacy behavior: if no output is specified, use the second argument as the output
+	if imageOpts.output == "" && len(args) == 2 {
+		output = args[1]
+		args = args[:1]
+	} else if imageOpts.output == "" || imageOpts.output == "-" {
```

Comment:
Changing the definition of positional args based on other flags is not something I want to do.

Reply:
I left that special case to not break scripts. The `export` subcommand seemed a good name for the task at hand, but using `save` for this task would be familiar from docker cli. The produced tar file would also fill the same use case and purpose.